warehouse/age-off/src/main/java/datawave/iterators/filter/ColumnVisibilityOrFilter.java

```diff
@@ -12,25 +12,23 @@
 public class ColumnVisibilityOrFilter extends TokenFilterBase {
```

Comment:
We may want to deprecate this class.  I think we tested https://github.com/NationalSecurityAgency/datawave/blob/integration/warehouse/age-off/src/main/java/datawave/iterators/filter/ColumnVisibilityTokenizingFilter.java and found it to be much faster.  Perhaps this caching change could be made to it.

Reply:
I can deprecate the ColumnVisibilityOrFilter and ColumnVisibilityAndFilter.

It looks like the ColumnVisibilityTokenizingFilter ultimately delegates to the `accept()` method found in the TokenFilterBase.